### PR TITLE
[ BB2-1452 ] Finishing changes before publish

### PR DESCRIPTION
**Pull request by dtisza1**
**JIRA Ticket:**
[BB2-1452](https://jira.cms.gov/browse/BB2-1452)

**User Story or Bug Summary:**

This PR adds a few finishing changes prior to publishing.

### What Should Reviewers Watch For?

If you're reviewing this PR, please check these things, in particular:

- TODO

### What Security Implications Does This PR Have?

Submitters should complete the following questionnaire:

- If the answer to any of the questions below is **Yes**, then here's a link to the associated Security Impact Assessment (SIA), security checklist, or other similar document in Confluence: N/A.
  - Does this PR add any new software dependencies? **No**.
  - Does this PR modify or invalidate any of our security controls?  **No**.
  - Does this PR store or transmit data that was not stored or transmitted before?  **No**.
- If the answer to any of the questions below is **Yes**, then please add StewGoin as a reviewer, and note that this PR should not be merged unless/until he also approves it.
  - Do you think this PR requires additional review of its security implications for other reasons? **No**.

### What Needs to Be Merged and Deployed Before this PR?
NOTHING.

This PR cannot be either merged or deployed until the following pre-requisite changes have been fully deployed:

- CMSgov/some_repo#42

I have gone through and verified that...:

- [X] This PR is reasonably limited in scope, to help ensure that:
  1. It doesn't unnecessarily tie a bunch of disparate features, fixes, refactorings, etc. together.
  2. There isn't too much of a burden on reviewers.
  3. Any problems it causes have a small "blast radius".
  4. It'll be easier to rollback if that becomes necessary.
- [X] I have named this PR and its branch such that they'll be automatically be linked to the (most) relevant Jira issue, per: <https://confluence.atlassian.com/adminjiracloud/integrating-with-development-tools-776636216.html>.
- [X] This PR includes any required documentation changes, including `README` updates and changelog / release notes entries.
- [X] All new and modified code is appropriately commented, such that the what and why of its design would be reasonably clear to engineers, preferably ones unfamiliar with the project.
- [X] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments, which include a JIRA ticket ID for any items that require urgent attention.
- [X] Reviews are requested from both:
  - At least two other engineers on this project, at least one of whom is a senior engineer or owns the relevant component(s) here.
  - Any relevant engineers on other projects (e.g. BFD, SLS, etc.).
- [X] Any deviations from the other policies in the [DASG Engineering Standards](https://github.com/CMSgov/cms-oeda-dasg/blob/master/policies/engineering_standards.md) are specifically called out in this PR, above.
  - Please review the standards every few months to ensure you're familiar with them.

### CODE REVIEW

File: README-bb2-dev.md [hunk 1]
```diff
@@ -30,4 +30,33 @@ Note: You may need to clean up already existing Docker containers, if you are ha
...
+## Installing cms-bluebutton-sdk from test.pypi.org
+
+The package can be installed from the test instance of the PyPI site for development testing. The default behavior is to install from the main site.
+
+To utilize the test.pypi.org repository use the following commands:
+
+To create and start the containers:
+
+```
+INSTALL_FROM_TEST_PYPI="True" docker-compose up -d
```
Comment (dtisza1) [inline README-bb2-dev.md:44]: @JFU-GIT @ajshred  Do you think adding this "INSTALL_FROM_TEST_PYPI" build argument would be good to add to the Node sample app as well?

Comment (dtisza1) [reply README-bb2-dev.md:44]: > INSTALL_FROM_TEST_PYPI

@JFU-GIT @ajshred   If we wanted to do something like this for the Node sample app, wondering if a better naming would be "BUILD_DEVELOPMENT"?

If you think this is better, I can change the argument name?

Comment (JFU-GIT) [reply README-bb2-dev.md:44]: I think it's good since both registries provide similar mechanism to let dev exercise publish operation and test with it.

**Review by JFU-GIT — APPROVED** [review]
Looks good, thanks!

File: README-bb2-dev.md (hunk 1 above)
Comment (ajshred) [reply README-bb2-dev.md:44]: Yes I would do it for both.  BUILD_DEVELOPMENT is a good name.

**Review by ajshred — APPROVED** [review]
Thanks Dave!  Looks good to me.

**Pull request merged**